charts/consent/templates/_consent.yaml.tpl

```diff
@@ -0,0 +1,63 @@
+{{- define "consent.config.yaml" -}}
+server:
+  type: simple
+  applicationContextPath: /
+  adminContextPath: /admin
+  connector:
+    type: http
+    port: 8080
+logging:
+  level: INFO
+  appenders:
+    - type: console
+      threshold: INFO
+      target: stdout
+    - type: sentry
+      dsn: https://foo:bar@sentry.io/baz {{/* Override on the command line */}}
+      threshold: ERROR
+      environment: {{ .Values.environment }}
+  loggers:
+    "org.reflections.Reflections": ERROR
+    "org.apache.pdfbox": ERROR
+database:
+  driverClass: org.postgresql.Driver
+  user: foo {{/* Override on the command line */}}
+  password: foo {{/* Override on the command line */}}
+  url: {{ .Values.databaseUrl }}
+  initialSize: 20
+  minSize: 20
+  maxSize: 128
+  validationQuery: SELECT 1
+elasticSearch:
+  servers:
+    - {{ .Values.elasticSearchServer1 }}
+    - {{ .Values.elasticSearchServer2 }}
+    - {{ .Values.elasticSearchServer3 }}
+  indexName: ontology
```

Comment:
is this left over from ontology or still needed here?

Reply:
Currently, Consent also talks to the ES servers. Ontology just does read-only stuff, Consent does index file updating. I want to extract that feature from Consent in the future, but for now, it's  necessary.